packages/editor/src/panels/scenes/SceneItem.tsx

```diff
@@ -26,37 +26,33 @@ import { PopoverState } from '@ir-engine/client-core/src/common/services/Popover
 import { deleteScene } from '@ir-engine/client-core/src/world/SceneAPI'
 import { StaticResourceType } from '@ir-engine/common/src/schema.type.module'
 import { timeAgo } from '@ir-engine/common/src/utils/datetime-sql'
-import { useClickOutside } from '@ir-engine/common/src/utils/useClickOutside'
-import { useHookstate } from '@ir-engine/hyperflux'
-import RenameSceneModal from '@ir-engine/ui/src/components/editor/panels/Scenes/modals/RenameScene'
+import RenameSceneModal from '@ir-engine/editor/src/panels/scenes/RenameSceneModal'
 import ConfirmDialog from '@ir-engine/ui/src/components/tailwind/ConfirmDialog'
+import { Popup } from '@ir-engine/ui/src/components/tailwind/Popup'
 import Button from '@ir-engine/ui/src/primitives/tailwind/Button'
 import Text from '@ir-engine/ui/src/primitives/tailwind/Text'
 import Tooltip from '@ir-engine/ui/src/primitives/tailwind/Tooltip'
-import { default as React, useEffect, useRef } from 'react'
+import { default as React } from 'react'
 import { useTranslation } from 'react-i18next'
 import { BsThreeDotsVertical } from 'react-icons/bs'
 import { LuTrash } from 'react-icons/lu'
 import { MdOutlineEdit } from 'react-icons/md'
-import { twMerge } from 'tailwind-merge'
 
 type SceneItemProps = {
   scene: StaticResourceType
-  moveMenuUp?: boolean
   handleOpenScene: () => void
   refetchProjectsData: () => void
   onRenameScene?: (newName: string) => void
   onDeleteScene?: (scene: StaticResourceType) => void
 }
 
-export const SceneItem = ({
+export default function SceneItem({
   scene,
-  moveMenuUp,
```

Comment:
moveMenuUp was used to fix some issue regarding context menu getting cut when screen size is small.

https://github.com/EtherealEngine/etherealengine/pull/10934
https://tsu.atlassian.net/browse/IR-3754

Reply:
right. we have changed it to [the `Popup` component](https://github.com/ir-engine/ir-engine/pull/125/files#diff-283abec6259bf451de484772618afba90537674c2460107affcd9b8d5470155aR93) which handles dynamic positioning